dpnp/dpnp_iface_trigonometric.py

```diff
@@ -1412,6 +1413,83 @@ def cumlogsumexp(
 )
 
 
+_LOGADDEXP2_DOCSTRING = """
+Calculates the logarithm of the sum of exponents for each element `x1_i`
+of the input array `x1` with the respective element `x2_i` of the input
+array `x2`.
```

Comment:
Do we need this part at all? the following text already explains it.

Reply:
I'm going to remove part of the text below. Or do you mean at line#1429?